### ISSUE TITLE
Command to import a component inside another

### ISSUE THREAD

**Issue by tanohzana** [tanohzana/reacli#22]
**Describe the solution you'd like**
I think it can be interesting to add a command that allows importing a component inside another.
For instance, if you are building a `house` component and you need to import a `door` component, you'd type:

`reacli import door in house`

In order to do that, we could use the `.reacli` file to store a dictionnary mapping components and paths.

Depends on #13

**Comment by anatol-grabowski** [comment]
I dont think this is a useful feature that worth the effort. As a user I would prefer to type a single import line into the file directly by myself instead of trying to remember one extra command and spend the same amount of time typing it.
